### PR TITLE
Helm changes for ndb-operator v0.0.6 release

### PR DESCRIPTION
**Pull request by shenoypritika**

### CODE REVIEW

File: charts/ndb-operator/values.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ image:
   repository: ghcr.io/nutanix-cloud-native/ndb-operator/controller
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "v0.0.5"
+  tag: "v0.0.6"
```
Comment (tuxtof) [inline charts/ndb-operator/values.yaml:7]: best is to let it empty in the value file

Comment (shenoypritika) [reply charts/ndb-operator/values.yaml:7]: Done

**Review by manavrajvanshi — APPROVED** [review]

File: charts/ndb-operator/crds/apiextensions.k8s.io_v1_customresourcedefinition_databases.ndb.nutanix.com.yaml [hunk 2]
```diff
@@ -107,6 +115,69 @@ spec:
                     description: Size of the database instance
                     minimum: 10
                     type: integer
+                  timeMachine:
+                    default: {}
+                    description: Information related to time machine that is to be
+                      associated with this database
+                    properties:
+                      dailySnapshotTime:
+                        default: "04:05:06"
```
Comment (akshmish) [inline charts/ndb-operator/crds/apiextensions.k8s.io_v1_customresourcedefinition_databases.ndb.nutanix.com.yaml:124]: I believe we have changed it to 04:00:00

**Comment by akshmish** [comment]
Can you confirm that this YAML file is generated by running the "make manifests" command on the pre-webhook main branch? I see some of the defaults are reverted to old ones (before webhook PR changed the defaults), so double-checking.

File: charts/ndb-operator/values.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@ image:
   repository: ghcr.io/nutanix-cloud-native/ndb-operator/controller
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "v0.0.5"
+  tag: ""
```
Comment (akshmish) [inline charts/ndb-operator/values.yaml:7]: Do you need v0.0.6 tag here?

Comment (tuxtof) [reply charts/ndb-operator/values.yaml:7]: no default is comming from appVersion

File: charts/ndb-operator/README.md [hunk 4]
```diff
@@ -98,7 +98,7 @@ spec:
       compute:
         id: ""
         name: ""
-      # A Software profile is a mandatory input for closed-source engines: MSSQL & Oracle
+      # A Software profile is a mandatory input for closed-source engines: MSSQL
```
Comment (tuxtof) [inline charts/ndb-operator/README.md:101]: why no more oracle ?

File: charts/ndb-operator/README.md [hunk 5]
```diff
@@ -129,7 +129,7 @@ The following table lists the configurable parameters of the NDB operator chart
 | `replicaCount`        | Number of replicas of the NDB Operator controller pods        | `1`                                                    |
 | `image.repository`    | Image for NDB Operator controller                             | `ghcr.io/nutanix-cloud-native/ndb-operator/controller` |
 | `image.pullPolicy`    | Image pullPolicy                                              | `IfNotPresent`                                         |
-| `image.tag`           | Image tag                                                     | `v0.0.5, defaults to Chart.appVersion if removed`      |
+| `image.tag`           | Image tag                                                     | `v0.0.6, defaults to Chart.appVersion if removed`      |
```
Comment (tuxtof) [inline charts/ndb-operator/README.md:132]: remove the version here and put

defaults to Chart.appVersion if empty

File: charts/ndb-operator/README.md (hunk 4 above)
Comment (shenoypritika) [reply charts/ndb-operator/README.md:101]: We don't support oracle. This was a mistake in the ReadMe.

**Review by tuxtof — APPROVED** [review]

**Pull request merged**